src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 0.3s linear;
```

Comment:
transition: 0.3s - it must be a variable. 
 Because 0.3s is repeated more than 1 time.